### ISSUE TITLE
sdl render job queue blocked on Mac

### ISSUE THREAD

**Issue by jimying** [pjsip/pjproject#2873]
### Describe the bug

Use sample vid_streamutil ,   test video streaming an avi file, app will block

### Steps to reproduce

1.  run vid_streamutil to streaming
     ./vid_streamutil --codec=h264 --remote=127.0.0.1:12345 --play-file ~/Downloads/a.avi
2.  No local render display; input 'q',  can't quit app

### PJSIP version

master (2.11-dev)

### Context

Macos v14.6 x86_64

### Log, call stack, etc

```shell
I add some debug log, find that, it block in:
vid_streamutil.c: 355  pjmedia_port_put_frame(play_file->renderer, &write_frame)
sdl_dev.c:944          job_queue_post_job(stream->sf->jq, put_frame, strm, 0, &status)

job_queue_post_job()  no return

I don't know how to fix it.  May be sdl render port is passive? aviplay run ok (which is active). 
But in windows platform,  vid_streamutil is ok
```

**Comment by jimying** [comment]
vid_streamutil should use pj_run_app( )  in main?  I will test it later

**Comment by jimying** [comment]
I have fix it, please see pr #2875
